hugo/content/8_release-updates/2023.md

```diff
@@ -7,6 +7,20 @@ weight: 802
 
 This section of the document gives you an overview of the new features and functions in rapyuta.io. It includes new features, improvements, announcements, deprecations, and bug fixes.
 
+## March 09
+
+| Feature | Type | Description |
+| ------- | ---- | ----------- |
+| [Fractional resources](/3_how-tos/34_networking-and-communication/ros-creating-routed-networks/) | New | You can now choose or add finer CPU and memory limits for ros networks. |
+| [JSON configurations](/3_how-tos/32_device-management/324_applying-configuration-on-devices/#creating-a-configuration-parameter-in-json-file-format) | New | You can now create a configuration parameter in JSON file format. | 
+| UI Improvements | Improvement | <ul><li>Catalog is now renamed to Packages</li>.<br> <li> You can now access the quick link to add secrets while adding a package with the executable type docker with a private image. </li></ul>|
+
+### Bug Fixes
+
+* The native rosbag jobs received the override yaml files generated for the custom rosbag jobs. This issue is now fixed.
```

Comment:
@shiv-mohith How can we improve this statement?

Here's what I came up with
```
1. Native rosbag jobs were being created with parameters identifiable only to custom rosbag jobs. This is now fixed.
```

Reply:
@pri-lk Let's keep it simple and generic without giving internal details

`Bug fixes related to rosbag creation`